.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+<!--
+Thank you for contributing to New Relic's Azure Functions integration! Before submitting your pull request, please review our contribution guidelines:
+
+* https://github.com/newrelic/newrelic-azure-functions/blob/master/README.md#contributing
+
+Following these best practices will help us review and merge your PR faster.
+
+When making updates to your PR, please add new commits instead of squashing. The PR will be squashed when merged.
+
```

Comment:
I would suggest to also ask not to force-push branches during a PR review. When contributors do so, comments no longer appear where they were originally placed.

I would also suggest to say something like: "When a reviewer includes a comment in your code, please answer to that comment stating whether you addressed it or you think it's best to do it in another way. This lets reviewer know whether you actually read that feedback, otherwise some comments may go unaddressed"